wrappers/python/simtk/openmm/app/amberprmtopfile.py

```diff
@@ -134,7 +134,8 @@ def __init__(self, file):
 
         atoms = list(top.atoms())
         for bond in prmtop.getBondsWithH():
-            top.addBond(atoms[bond[0]], atoms[bond[1]])
+	        if bond[0] < len(list(top.atoms())):
+				top.addBond(atoms[bond[0]], atoms[bond[1]])
```

Comment:
Just a few comments -- the indentation here looks wrong (maybe a mix of tabs and spaces?).  This will break with Python 3, since indentation within a file must be consistent. (It may not work with Python2 either, I'm not sure)

Also, for every bond you loop through, you construct a list from `top.atoms()` and then immediately throw it away.  This is very inefficient.  It would be better to store `len(list(top.atoms))` before the loop and used that cached value.